README.md

```diff
@@ -14,7 +14,9 @@ Unvanquished is an arena game with RTS elements (you can build) in which 2 very
 
 This repository contains the gamelogic of the game Unvanquished.
 
-If you just want to play the game, then technically you only need to build the [Dæmon engine](https://github.com/DaemonEngine/Daemon) and [download the game's assets](#downloading-the-games-assets).
+The game provide up-to-date downloads on the [download page](https://unvanquished.net/download/).
```

Comment:
`The game provide` doesn't make much sense, how about `We provide`

Reply:
Ah yes, French people use and abuse of [metonymy](https://en.wikipedia.org/wiki/Metonymy]). I leave some lying around everywhere. 😂️